scripts/globals/spells/dia.lua

```diff
@@ -46,15 +46,19 @@ spell_object.onSpellCast = function(caster, target, spell)
     -- Check for Bio
     local bio = target:getStatusEffect(xi.effect.BIO)
 
-    -- Do it!
-    target:addStatusEffect(xi.effect.DIA, 1 + dotBonus, 3, duration, 0, 10, 1)
-    spell:setMsg(xi.msg.basic.MAGIC_DMG)
-
     -- Try to kill same tier Bio (non-default behavior)
     if BIO_OVERWRITE == 1 and bio ~= nil then
         if bio:getPower() == 1 then
             target:delStatusEffect(xi.effect.BIO)
         end
+    else
```

Comment:
If this allows overwrite, then it deletes proper bio tier... but then it does nothing else, not even damage.

Reply:
And moved this around to make it do what it was supposed to do.